core/modules/field_ui/field_ui.admin.inc

```diff
@@ -816,11 +816,77 @@ function field_ui_field_overview_form_submit($form, &$form_state) {
  * Form constructor for the field display settings for a given view mode.
  *
  * @see field_ui_menu()
+ * @ingroup forms
+ */
+function field_ui_display_overview($entity_type, $bundle) {
+  if (is_object($bundle)) {
+    $bundle_label = $bundle->name;
+  }
+  $bundle = field_extract_bundle($entity_type, $bundle);
+  $admin_path = _field_ui_bundle_admin_path($entity_type, $bundle);
+
+  $entity_info = entity_get_info($entity_type);
+  if (!isset($bundle_label)) {
+    $bundle_label = $entity_info['label'];
+  }
+  $view_mode_settings = field_view_mode_settings($entity_type, $bundle);
+  $custom_view_modes = config_get('entity.view_modes', 'view_modes');
+  $view_modes = $entity_info['view modes'];
+  if (is_array($view_modes)) {
+    backdrop_sort($view_modes, array('label' => SORT_STRING));
+  }
+
+  $custom = array();
+  $default = array();
+  // Sort view modes into active/default, and merge in view mode settings.
+  foreach ($view_modes as $view_mode_name => $mode) {
+    $custom_settings = isset($view_mode_settings[$view_mode_name]['custom_settings']) ? (bool) $view_mode_settings[$view_mode_name]['custom_settings'] : (bool) $mode['custom settings'];
+
+    $custom[$view_mode_name] = array(
+      'label' => $mode['label'],
+      'custom settings' => $custom_settings,
+      'storage' => !empty($custom_view_modes[$entity_type][$view_mode_name]) ? 'Custom' : 'Default (module-provided)',
+      'status' => $custom_settings ? 'Customized' : 'Using the <em>Default display</em>',
+    );
+
+    if (!($custom_settings)) {
+      $default[$view_mode_name] = $custom[$view_mode_name];
+      unset($custom[$view_mode_name]);
+    }
+  }
+
+  $build = array(
+    '#entity_type' => $entity_type,
+    '#bundle' => $bundle,
+    '#attached' => array(
+      'css' => array(backdrop_get_path('module', 'field_ui') . '/css/field_ui.css'),
+    ),
+  );
+  $build['help'] = array(
+    '#type' => 'markup',
+    '#markup' => '<p>' . t('When @label items are viewed alone on a full page they are often displayed differently than when viewed as part of a list. Each way an item can be displayed is called a <strong>View Mode</strong>.', array('@label' => $bundle_label)) . '</p>',
+  );
+  $build['view_modes'] = array(
+    '#type' => 'group',
```

Comment:
I don't think `#type = group` exists? We can probably remove this `#type` without any negative impact.